MaterialLib/SolidModels/LinearElasticIsotropic.h

```diff
@@ -0,0 +1,104 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATERIALLIB_SOLIDMODELS_LINEARELASTICISOTROPIC_H_
+#define MATERIALLIB_SOLIDMODELS_LINEARELASTICISOTROPIC_H_
+
+#include "SolidsBase.h"
+
+namespace Solids
+{
+template <int DisplacementDim>
+class LinearElasticIsotropic final : public SolidsBase<DisplacementDim>
+{
+public:
+    /// Variables specific to the material model
+    class MaterialProperties
+    {
+        using P = ProcessLib::Parameter<double>;
+        using X = ProcessLib::SpatialPosition;
+
+    public:
+        MaterialProperties(P const& youngs_modulus, P const& poissons_ratio)
+            : _youngs_modulus(youngs_modulus), _poissons_ratio(poissons_ratio)
+        {
+        }
+
+        double lambda(double const t, X const& x) const
```

Comment:
This just the interface of Parameter class. Would be writing 0 instead of time otherwise, but that is even more questionable.